packages/domain/src/use-cases/profile/FollowProfile.ts

```diff
@@ -17,19 +20,21 @@ import {
 
 export type FollowRequestFee = {
   amount: Amount<Erc20>;
-  contractAddress: string;
+  contractAddress: EvmAddress;
   recipient: EvmAddress;
 };
 
 export type UnconstrainedFollowRequest = {
```

Comment:
"Unconstrained" is a term born from the need of make a distiction between "free" and "free with follower only" follow module. This because Proxy Actions could only process "free" follows.

I think we should simplify here between "paid" and "free".

Reply:
done